GooglePlayInstant/Editor/AndroidManifest/LegacyAndroidManifestUpdater.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+using System.Xml.Linq;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor.AndroidManifest
+{
+    /// <summary>
+    /// An IAndroidManifestUpdater for Unity versions 2017 and earlier that saves manifest changes to
+    /// the file Assets/Plugins/Android/AndroidManifest.xml
+    /// </summary>
+    public class LegacyAndroidManifestUpdater : IAndroidManifestUpdater
```

Comment:
nit: `LegacyAndroidManifestUpdater` may be a little confusing, I would have expected that this was a deprecated behavior that is now being slowly phased out. Maybe `AndroidManifestUpdaterPre2017` or something similar?

Reply:
In a way, it is "being slowly phased out". Unity 2018 is the latest version and usage of older versions (5.x, 2017.x) will be phased out; we'll eventually delete this implementation. I think "Legacy" is a common prefix for this sort of situation.

I'm open to other names, but haven't thought of one I like. Something like Pre2018AndroidManifestUpdater would be correct, but seems awkward to me personally.

I'd prefer to leave as-is unless you feel strongly.